### DISCUSSION TITLE
Raspberry Pi?

### DISCUSSION THREAD

**Discussion by roundmoundofsound** [marin-m/SongRec#28] [Q&A]
Hi there, am I right to assume that (a) I could run this on a raspberry pi, and (b) I could pass the current track information to another application? I would like to display the current track on an e-ink display, rather than using the GUI.

**Comment by schworak** [comment]
I had this working great when I installed it about a year ago. But then this week I had to rebuild my PI and now I can't get the PPA installed. So yes it will work and I always used the command line interface. But it did work well on the PI. (until now)

**Reply by TheMopCloset** [reply]
Hi! Did you ever get this working? I'd like to somehow display just the artwork on a small monitor. I want to be able to listen to my records or cassettes, and have the album artwork display on a monitor. This is the closest I've been able to get to that, but do you know what Raspberry Pi would work? Or how to get only the artwork?

**Reply by schworak** [reply]
No, I never got it working on the PI. Ultimately, I got it working on another machine running Ubuntu, so i simply gave up on the PI. I was trying to use the PI as a music player but that seemed to be too much for it at times as it would over heat and throttle and make the music skippy. So I ended up doing a full Ubuntu install on a real computer (low power) and did it there. SongRec works great there though.

**Comment by GoaSkin** [comment]
I run Songrec on a Raspberry Pi 5 in a Neo One V5 case. 

Songrec is usable with a few limitations:

- I need to use the "plughw" ALSA device. If I use the "hw" device, Songrec recognizes a lot of garbage.
- For some reasons, songrec my slow down when running a while and also exhause the available RAM before it is terminated by the OOM-killer. Therefore, I run songrec as a systemd service with a limit of 100MB RAM. This doesn't prevent the process from being killed but avoids the system from being freezed before.
